subprojects/cairo.wrap

```diff
@@ -0,0 +1,5 @@
+[wrap-git]
+directory=cairo
+url=https://github.com/ebraminio/cairo.git
```

Comment:
needed to apply https://github.com/ebraminio/cairo/commit/e6f69054671d4c004cb1f0be4a1edbb4c4c1810d to make it work

Reply:
some other hard coding done on https://github.com/ebraminio/cairo/commit/c30fa2f0893527af1f7a41bd6f9ac6fc10e99e27 as the options weren't yield: 'true' https://mesonbuild.com/Build-options.html#yielding-to-superproject-option (at least explicitly?)